adafruit_connection_manager.py

```diff
@@ -128,8 +128,28 @@ def get_radio_socketpool(radio):
         elif class_name == "WIZNET5K":
             import adafruit_wiznet5k.adafruit_wiznet5k_socket as pool  # pylint: disable=import-outside-toplevel
 
-            # Note: SSL/TLS connections are not supported by the Wiznet5k library at this time
-            ssl_context = create_fake_ssl_context(pool, radio)
+            # Note: At this time, SSL/TLS connections are not supported by older
+            # versions of the Wiznet5k library or on boards withouut the ssl module
+            # see https://docs.circuitpython.org/en/latest/shared-bindings/support_matrix.html
+            ssl_context = None
+            try_ssl = False
+            cp_version = sys.implementation[1]
+            if pool.SOCK_STREAM == 1 and cp_version[0] >= 9:
```

Comment:
This was my best thought on checking the version. If there is a better way, let me know (I also hope ssl everywhere doesn't go out in a revision...)